Source/Settings.as

```diff
@@ -210,6 +210,15 @@ vec4 Setting_Gearbox_TextColor = vec4(1, 1, 1, 1);
 string Setting_Gearbox_Font = "DroidSans.ttf";
 
 
+enum AccelerationUnit
+{
+	MetersPerSecondPerSecond,
+	KilometersPerHourPerSecond
+}
+
+[Setting category="Acceleration" name="Acceleration unit of measurement" description="Make sure to change max acceleration, for km/h/s = 54, for m/s/s = 15"]
```

Comment:
Is there a reason why `Setting_Acceleration_MaximumAcceleration` is not scaled automatically?

Reply:
Seems like that would be pretty astonishing behaviour, to have the user have a setting that has one value that is scaled by 3.6 if you switch the unit of measurement, in that if that happens you don't really know what the value actually is. 

I think instead I'll rename the setting and add another one for km/h/s so users don't need to adjust anything.

Reply:
New commit tries to fix this.